xain_fl/config.py

```diff
@@ -21,7 +21,7 @@
 
 def init_config():
     """Creates initial config file if non exists"""
-    logger.info("Initializing config in %s", config_file)
+    logger.info(f"Initializing config in {config_file}")
```

Comment:
Please do not use f-strings in logs, it's a rule in python, according to https://www.python.org/dev/peps/pep-0498/